### PR TITLE
doc: fix yaml syntax

### PR DESCRIPTION
**Pull request by mhor**
see [https://github.com/symfony/symfony-standard/issues/426](https://github.com/symfony/symfony-standard/issues/426)

### CODE REVIEW

File: Resources/doc/config.rst [hunk 1]
```diff
@@ -306,15 +306,13 @@ It is possible to connect to several mongodb servers on one connection if
 you are using a replica set by listing all of the servers within the connection
 string as a comma separated list.
 
-.. configuration-block::
```
Comment (malarzm) [inline Resources/doc/config.rst]: Why are we ditching `configuration-block`?

Comment (mhor) [inline Resources/doc/config.rst]: IMO this is an useless block, we had only yaml under this block

Comment (malarzm) [inline Resources/doc/config.rst]: I think I'd rather keep them and add other configurations instead (in another PR)

Comment (mhor) [inline Resources/doc/config.rst]: I have reverted changes for  `configuration-block`

**Comment by malarzm** [comment]
Thanks @mhor!

**Pull request merged**